src/review/views.py

```diff
@@ -1175,21 +1166,26 @@ def add_review_assignment(request, article_id):
                     )
                 )
 
+    display_past_reviewers = request.journal.get_setting(
+        'general',
+        'display_past_reviewers')
+    enable_suggested_reviewers = request.journal.get_setting(
+        'general',
+        'enable_suggested_reviewers',
+    )
+    if display_past_reviewers and enable_suggested_reviewers:
+        table_sort = [7, 8]
+    elif display_past_reviewers or enable_suggested_reviewers:
+        table_sort = [7]
```

Comment:
What are these magic numbers? presumably numbered columns? Since the sorting happens on the client only, maybe the template should be doing this check so the logic is not split between the two. I feel like it will be easy to miss here if we later enhance that table.

Otherwise, have you considered sorting the queryset by the annotated fields instead?